### PR TITLE
made improvements in exam module

### PR DESCRIPTION
**Pull request by Arpit-SerpentCS**

### CODE REVIEW

File: exam/models/exam.py [hunk 1]
```diff
@@ -245,14 +241,6 @@ def set_cancel(self):
...
         self.write({'state': 'cancelled'})
 
-    @api.multi
-    def _validate_date(self):
-        '''Method to check start date should be less than end date'''
-        for exm in self:
-            if exm.start_date > exm.end_date:
-                return False
-        return True
-
```
Comment (JayVora-SerpentCS) [inline exam/models/exam.py]: any reason for removing? Everytime you make PR, better you mention what activities are done

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Improve the README.rst file of a module. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/README.rst

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please remove `mail` module from dependency. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/__manifest__.py#L13

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Add domain in inside the search condition if it is used one time. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/wizard/move_standards.py#L21, L31, L47

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please remove `-` from groups. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/wizard/subject_result.xml#L44

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Please add this line directly in dictionary. 
result_data.append({'subject': subject.subject_id and subject.subject_id.name or ''
https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/report/add_exam_result.py#L20

list_result.append({'standard_id': sub_id.standard_id.standard_id.name,
https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/report/result_info.py#L25

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS Remove line no 5 and add "_" in 3rd line. https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/report/result_info.py#L5

**Comment by murtuzasaleh** [comment]
@Arpit-SerpentCS No need of data if noupdate="0". https://github.com/Arpit-SerpentCS/OdooEduERP/blob/11.0_exam/exam/demo/exam_demo.xml#L4

**Comment by Arpit-SerpentCS** [comment]
@murtuzasaleh Please review the code. Made improvements and tested according to the comments.

cc @JayVora-SerpentCS

**Pull request merged**